shared.bazelrc

```diff
@@ -1,3 +1,5 @@
+startup --output_base=bazel-output-base
+
```

Comment:
This will change the output_base for developers as well as CI. If we want this for CI, should we have a CI-specific bazelrc?

Reply:
We need it for developers as well, as we need the fixtures to match.